lib/type_handler_common.py

```diff
@@ -148,7 +148,8 @@ def process_structure_info(
                     structure_svc,
                     enum_svc,
                     ref_path)
-            new_type['properties'].setdefault(field.name, newprop)
+            #new_type['properties'].setdefault(field.name, newprop)
```

Comment:
I think this comment is not needed anymore.

Reply:
Thank you for spotting this.